### PR TITLE
[WIP] Added BooleanFacet component and its unit tests

### PR DESCRIPTION
**Pull request by Jfelix61**
## Description
As per title.
## List of changes
- Added BooleanFacet component along with its unit tests
## Associated Github Issues
#108

### CODE REVIEW

File: examples/sandbox/src/App.js [hunk 1]
```diff
@@ -230,6 +231,11 @@ export default function App() {
                         label="Acres"
                         view={SingleSelectFacet}
                       />
+                      <Facet
```
Comment (JasonStoltz) [inline examples/sandbox/src/App.js]: There is already a `Facet` for this field further up in this file. Can you remove this and change the view of the existing facet?

Comment (Jfelix61) [reply examples/sandbox/src/App.js]: Sure!

**Comment by JasonStoltz** [comment]
Hey @Jfelix61! Our designer, @brianearwood, added some commits to your work to add styles. Can you review them in this PR and cherry-pick them into your branch if you are 👍 with them?

**Comment by Jfelix61** [comment]
> Hey @Jfelix61! Our designer, @brianearwood, added some commits to your work to add styles. Can you review them in this PR and cherry-pick them into your branch if you are  with them?

Great! I'll be adding them later.

**Comment by elasticcla** [comment]
Hi @Jfelix61, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in your Git [commit](https://github.com/elastic/search-ui/pull/376.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by Jfelix61** [comment]
Created new cleaner PR #383

**Pull request closed without merging**